### ISSUE TITLE
zookeeper miss changes notification 

### ISSUE THREAD

**Issue by sstong123** [kelseyhightower/confd#713]
command:
`confd-0.16.0-windows-amd64.exe -watch -backend zookeeper -node 127.0.0.1:2181 --log-level debug`

but missing the changes notification when the watched node updated.

**Comment by sstong123** [comment]
@okushchenko ,how is it going on?

**Comment by sstong123** [comment]
@okushchenko any update?

**Comment by okushchenko** [comment]
@valodzka yes, it looks like it. The changes that I propose in the #568 should fix this issue by introducing a safer way for backends to watch for updates.
